docs/getting-started.md

```diff
@@ -211,10 +254,13 @@ The cause of the _overscanning_ can be found in the [repository field of acorn's
 },
 ```
 
-One can not fix the sources of already released package artifacts so to resolve the missing _directory_ in the metadata of `NPM::acorn:7.1.1` you can use a [VCS path curation][ort-curations]:
+Although it is not possible to fix the sources of previously released package artifacts, you can add the missing _directory_ in the metadata
```

Comment:
maybe: 

you can add the missing _directory_ in the metadata -> you can curate the metadata to specify the directory containing the sources of the package

Reply:
I've edited this text again.